src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_tls_certs.h

```diff
@@ -0,0 +1,101 @@
+#define GLOBALSIGN_ROOT_CERT "-----BEGIN CERTIFICATE-----\n\
```

Comment:
oh, I answered this question already, maybe on the main page of the PR, but apparently it is not here.

For now I am satisfied with these certs here. I've asked question regarding using multiple certs in a single array, we'll get our answers or if you dare JC you could experiment it.